client/src/components/Conversations/Convo.tsx

```diff
@@ -48,31 +46,20 @@ export default function Conversation({ conversation, retainView, toggleNav, isLa
 
     // set conversation to the new conversation
     if (conversation?.endpoint === EModelEndpoint.gptPlugins) {
-      let lastSelectedTools = [];
-      try {
-        lastSelectedTools = JSON.parse(localStorage.getItem('lastSelectedTools') ?? '') ?? [];
-      } catch (e) {
-        // console.error(e);
-      }
+      const lastSelectedTools = JSON.parse(localStorage.getItem('lastSelectedTools') ?? '') ?? [];
```

Comment:
Please wrap JSON.parse with a try/catch block

Reply:
I fixed and restored some things in the commit: d26c2c16be6779806a776c93c2601e3a21c59021